src/rest.cpp

```diff
@@ -33,7 +33,7 @@ enum class RetFormat {
 };
 
 static const struct {
-    enum RetFormat rf;
+    enum class RetFormat rf;
```

Comment:
Fails on clang with:

```
rest.cpp:36:10: error: reference to scoped enumeration must use 'enum' not 'enum class'
    enum class RetFormat rf;
         ^~~~~~
rest.cpp:71:13: error: reference to scoped enumeration must use 'enum' not 'enum class'
static enum class RetFormat ParseDataFormat(std::string& param, const std::string& strReq)
            ^~~~~~
2 errors generated.

Reply:
argh should have gone with my first version (just removing enum altogether!).

I'll update the PR soon.
Thanks!